docker-compose-prod.yml

```diff
@@ -0,0 +1,34 @@
+version: '2'
```

Comment:
According to the explanation for this on the [docker site](https://docs.docker.com/compose/extends/), it would make sense to have 

docker-compose.yml for the common stuff
docker-compose.override.yml for the dev-specific stuff (development server)
docker-compose.prod.yml for the prod-specific stuff

Pretty sure it would end up being less redundant, what do you think?